Add rimgu redirect support for imgur

3nprob opened the pull request:
imgur.com is particularly overwhelming with tracking scripts and 3rd-party requests. A new project that fills the gap for embeddeds as well as web URLs is https://codeberg.org/3np/rimgu.

This adds imgur/rimgu support to Privacy Redirect.

MonsieurBibo committed:
Fixed french typo in store.md

Commit:
Add rimgu redirect support for imgur

EgosOwn commented:
Hi there, I recently made an imgur front end too (only a few days older than 3nprob's). I made it with privacy redirect in mind so i would be disappointed if I couldn't get it integrated, but rimgu looks good too and has more features.3nprob, maybe you want to collaborate? I don't know if dual projects would be accepted for 1 site in this extension, but i wouldn't have a problem with it. 

Anyone have thoughts?

https://imgin.voidnet.tech/
https://git.voidnet.tech/kev/imgin

3nprob commented:
> Hi there, I recently made an imgur front end too (only a few days older than 3nprob's). I made it with privacy redirect in mind so i would be disappointed if I couldn't get it integrated, but rimgu looks good too and has more features.3nprob, maybe you want to collaborate? I don't know if dual projects would be accepted for 1 site in this extension, but i wouldn't have a problem with it.
> 
> Anyone have thoughts?
> 
> https://imgin.voidnet.tech/ https://git.voidnet.tech/kev/imgin

Convergent evolution eh :D

Just my 5c: I always found it a bit unfortunate that PrivacyRedirect refers to the re-implementation projects names ("Bibliogram redirects" rather than "Instagram redirects"). As e.g. Piped or Yotter can be used just fine as "Invidious". I assume this is to avoid trademark/copyright cases a la https://www.techspot.com/news/91650-facebook-bans-sends-cease-desist-letter-developer-unfollow.html.

I'll take a look at your project as well - regardless of if they stay separate or merge I think as long as they're separate we should try to keep them behaving identically from an API perspective so they should be interchangeable in Privacy Redirect.

miku86 commented:
> Hi there, I recently made an imgur front end too (only a few days older than 3nprob's). I made it with privacy redirect in mind so i would be disappointed if I couldn't get it integrated, but rimgu looks good too and has more features.3nprob, maybe you want to collaborate? I don't know if dual projects would be accepted for 1 site in this extension, but i wouldn't have a problem with it.
> 
> Anyone have thoughts?
> 
> https://imgin.voidnet.tech/ https://git.voidnet.tech/kev/imgin

Hey there,

great work! 

There is a discussion about the naming: #302.
IMHO it doesn't matter which application/frontend we can use as long as it does the job.

I think we should keep the extension open to all kind of implementations that do the job, 
e.g. we already use both libreddit and teddit as redirects.

xanoni committed:
Remove Onion v2 links ...

... because everything comes to an end

xyzshantaram committed:
Clarify README

This was recently posted on Hacker News, and someone wrote that it wasn't clear enough that you'd be accessing the same content. Hopefully this commit fixes that.

Commit:
Added old.reddit.com to targets, removed old.reddit.com/i.reddit.com
from redirects.

SimonBrazell commented:
Looks great @3nprob I'll have a play with it now.

SimonBrazell commented:
Not sure I can push this live without at least one instance though, people will enable it and wonder why it's not working.

ghost commented:
~~It's not "official" by any means, but I have https://i.actionsack.com up as an rimgu instance.~~

~~Docker container: https://github.com/xnaas/docker-rimgu/pkgs/container/rimgu~~

~~I've [opened a number of issues](https://codeberg.org/3np/rimgu/issues) in the [source repository](https://codeberg.org/3np/rimgu) as well to maybe inspire some more action, an instance list, etc.~~

~~Edit: I host a [number of services with decent uptime](https://status.actionsack.com/). 😃~~

---

Edit (2022-04-01): Moved to [rimgo](https://codeberg.org/video-prize-ranch/rimgo) instead. [My periodically+automatically updating docker container for it.](https://ghcr.io/xnaas/rimgo)

Commit:
add platform.twitter.com

vymague commented:
How does it compare with rimgo? That one has an instance list. https://codeberg.org/video-prize-ranch/rimgo#instances
